docs/conversion_examples_gallery/conversion_example_gallery.rst

```diff
@@ -70,6 +70,7 @@ Imaging
     Miniscope <imaging/miniscope>
     Scanbox <imaging/scanbox>
     ScanImage <imaging/scanimage>
+    ScanImage Multi-File <imaging/multifilescanimage>
```

Comment:
For the conversion gallery, I think it makes more sense to combine multifilescanimage and single file scan image in a single page with different headings. So, instead of two different pages for scan image with two subheadings each, we would have:
- Convert single plane single file imaging data
- Convert multi-plane (volumetric) single file imaging data
- Convert single plane multi-file (buffered) imaging data
- Convert multi-plane (volumetric) multi-file (buffered) imaging data